Source/Scene/Scene.js

```diff
@@ -597,7 +597,7 @@ define([
          */
         pickPositionSupported : {
             get : function() {
-                return this._context.depthTexture;
+                return defined(this._context.depthTexture) && defined(this._globeDepth);
```

Comment:
Are you sure about this change?  `context.depthTexture` should return a boolean and `_globeDepth` should be defined if `context.depthTexture` is true.

See the `Scene` constructor and implementation for `context.depthTexture`.